solo/methods/base.py

```diff
@@ -506,21 +512,34 @@ def validation_step(self, batch: List[torch.Tensor], batch_idx: int) -> Dict[str
                 the classification loss and accuracies
         """
 
-        X, targets = batch
-        batch_size = targets.size(0)
+        if dataloader_idx == self.val_loader_keys.index("augmented"):
+            _, X, targets = batch
```

Comment:
I think this line can be moved out of the if and replaced with something like `*_, X, targets = batch`